submissions/fabunmiTolulope/dom/productPage.js

```diff
@@ -0,0 +1,59 @@
+let quantity = document.getElementById('quantity');
+let price = document.getElementById('price');
+let shippingCost = document.getElementById('shippingCost');
+let totalCost = document.getElementById('totalCost');
+let submit = document.getElementById('btn');
+let pricePerUnit = price.innerHTML;
+let productPrice = pricePerUnit * quantity.value;
+let quantityRegex = /^[1-9]+[0-9]*$/;
+
+function quantityValue(){
+    if (!(quantity.value.match(quantityRegex))) {
+        submit.disabled = true;
+        quantity.nextElementSibling.innerHTML = "Please enter a valid quantity";
+    } else{
+        submit.disabled = false;
+        quantity.nextElementSibling.innerHTML = "";
+    }
+}
+
+function priceValue() {
+        if (quantity.value > 0) {
```

Comment:
Format this line of code

Reply:
Hi @adaugochi how do I format a line of code?